dotnet-base-libraries/CryptographyRandomNumberGenerator/App/CryptographicHelpers.cs

```diff
@@ -0,0 +1,14 @@
+using System.Security.Cryptography;
+
+namespace App;
+
+public static class CryptographicHelpers
```

Comment:
I like the renaming we did with the benchmarks. I think it would be good to go ahead and do the same here. Also, as was pointed out by @AndreCarvalho, since we are not specifying a seed with `RandomNumberGenerator`, we should probably be consistent and just use `Random.Shared` for the general random number case. With that change we can drop the static member variable as well, which is nice.
Then for the rename we can just do something like `GenerateSecureRandomKey`, `GenerateSecureRandomInteger`, `GenerateGeneralRandomInteger`